_posts/docker/2023-08-09-docker-chapter3.md

```diff
@@ -0,0 +1,175 @@
+---  
+title: Docker  시작
+layout: post   
+image : /assets/img/study/docker/book-logo.jpg
+categories : [docker]
+customexcerpt: 도커로 작업 시작하기!
+---
+
+<span class = "alert g">작성자 : 정지우</span>
+
+<!-- 아래 2줄은 목차를 나타내기 위한 심볼이니 건들지 말아 주세요 -->
+* random line to make it work. This will be removed.
+{:toc} 
+
+
+
+1. 단순한 형태의 컨테이너 시작 & 이용
```

Comment:
이것이 목차를 의미하는 경우

위 코드 
~~~
<!-- 아래 2줄은 목차를 나타내기 위한 심볼이니 건들지 말아 주세요 -->
* random line to make it work. This will be removed.
{:toc}
~~~
에서 목차를 생성하기 때문에 혼동할 수 있어 보입니다.